e2e-tests/playwright/tests/functional/system_console/system_users/export_data.spec.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect} from '@playwright/test';
+
+import {test} from '@e2e-support/test_fixture';
+import {duration} from '@e2e-support/util';
+import {components} from '@e2e-support/ui/components';
+
+test('MM-T5522 Should begin export of data when export button is pressed', async ({pw, pages}) => {
+    test.slow();
```

Comment:
intentional ?

Reply:
yes discussed in https://github.com/mattermost/mattermost/pull/26077#discussion_r1471896573